apps/participants/utils.py

```diff
@@ -0,0 +1,28 @@
+from challenges.models import Challenge
+
+from .models import Participant
+
+
+def check_user_belongs_to_participant_team(user, participant_team_id):
+    """Returns boolean if the user belongs to the participant team or not"""
+    return Participant.objects.filter(user=user, team__id=participant_team_id).exists()
+
+
+def check_participant_team_participated_in_challenge(participant_team_id, challenge_id):
+    """Returns boolean if the Participant Team participated in particular Challenge"""
+    return Challenge.objects.filter(pk=challenge_id, participant_team__id=participant_team_id).exists()
+
+
+def check_user_participated_in_challenge(user, challenge_id):
+    """Returns boolean if the user has participated in a particular challenge"""
+    participant_teams = Participant.object.filter(user=user).values_list('teams', flat=True)
+    return Challenge.objects.filter(pk=challenge_id, participant_teams__in=participant_teams).exists()
```

Comment:
This should essentially use the information from get_participant_team_id_of_a_user_for_a_challenge function. 
If participant_team is provided, use the info otherwise call the function to get that info based on user. 
Should also rename these functions to make them concise.